custom_components/elasticsearch/system_info.py

```diff
@@ -19,8 +21,8 @@ async def async_get_system_info(self):
             system_info = await self._hass.helpers.system_info.async_get_system_info()
 
             # see homeassistant/helpers/system_info.py in main home-assistant repo
-            if self._hass.components.hassio.is_hassio():
-                hostname = self._hass.components.hassio.get_host_info().get("hostname", "UNKNOWN")
+            if system_info.get("hassio"):
+                hostname = get_host_info(self._hass).get("hostname", "UNKNOWN")
```

Comment:
maybe not for this PR but we should consider not including "UNKNOWN" when the host name can't be found. perhaps just not publishing the field

Reply:
Ah yeah thanks for the reminder. I've attempted to address in https://github.com/legrego/homeassistant-elasticsearch/pull/270/commits/d9217ab1be2f50f27ddd5a45d35e44e8f9858a21, let me know what you think.